### DISCUSSION TITLE
Permission view people

### DISCUSSION THREAD

**Discussion by GandonNicolas** [sulu/sulu#7002] [Q&A] [answered]
I would like to assign to a role, the permission to only view `contacts -> people` but it doesn't work, is this normal behaviour? 

Step to reproduce :

1. On https://sulu.rocks/admin/#/
2. I login with admin. I create a 'test' role with the permission to view contacts. 
3. I create a user and assign him the 'test' role. 
4. I log out of the admin account and log in with the new user. 
5. The contact menu does not appear.

**Comment by NeuralClone** [comment, marked as answer]
I think this is working as intended. If you look at [ContactAdmin](https://github.com/sulu/sulu/blob/2.5/src/Sulu/Bundle/ContactBundle/Admin/ContactAdmin.php) and how permissions are checked, users need to have the `EDIT` permission to access Contacts:

https://github.com/sulu/sulu/blob/099163a96ba83b84a7ec8ca3986e3e9a8bc097e7/src/Sulu/Bundle/ContactBundle/Admin/ContactAdmin.php#L97-L107

Other permissions are then required to perform various actions. All of the bundles are set up this way.

You have a bit more flexibility with media collections since you can grant `EDIT` access to the section and then limit access on individual collections.

I'm not aware of an easy way to do this with contacts.

**Comment by GandonNicolas** [comment]
That clarifies it for me 🙏  Thank you for your answer. @NeuralClone
